modules/store/src/store.ts

```diff
@@ -19,17 +18,20 @@ export class Store<T = object>
   extends Observable<T>
   implements Observer<Action>
 {
-  private readonly state: Signal<T>;
+  /**
+   * @internal
+   */
+  readonly state: Signal<T>;
 
   constructor(
-    state$: StateObservable,
+    readonly state$: StateObservable,
```

Comment:
after applying the suggested change for `MockState`, this change can be reverted

```suggestion
    state$: StateObservable,
```